components/Downloads/DownloadHeader/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { FormattedMessage, useIntl } from 'react-intl';
+import styles from './index.module.scss';
+import SectionTitle from '../../Common/SectionTitle';
+import type { NodeReleaseData } from '../../../types';
+import type { FC } from 'react';
+
+type DownloadHeaderProps = {
```

Comment:
Please make this type `non-optional`.

Reply:
It doesn't make sense for the release data of a DownloadHeader to be nullable

Reply:
Yep sure IDK why on nodejs.dev we do that (I have search on issue/pr and I found nothing 🤗)

Reply:
nodejs.dev is a different repo with a different story and probably these things were not documented.